app/Models/Post.php

```diff
@@ -55,4 +57,12 @@ public function reactions()
     {
         return $this->hasMany(Reaction::class);
     }
+
+    /**
+     * Get all fo the tags for the post.
+     */
+    public function tags()
```

Comment:
I don't think we need this? because the package comes with functionality to grab the tags associated with a post already with `tagNames()`